pluggin/consts.py

```diff
@@ -1,16 +1,18 @@
 import os
+from pathlib import Path
 
-from sqlalchemy import create_engine
 
-LLM_CONFIG_COLUMN = "llm model"
-DEAFULT_LANGUAGE = "pt-BR"
-
-
-class SPEACH_C:
+class AZAZEL_STONE:
+    DEAFULT_LANGUAGE = "pt-BR"
+    FOLDER = Path().cwd()
+    TEMP = "/var/tmp"
+    PLUGGIN_PATH = FOLDER / "pluggin"
+    OUTPUT_FILE = Path(f"{TEMP}/output.wav")
+    IMAGES = PLUGGIN_PATH / "images"
     ACTIVATION_PHRASE = "Azazel"
-    FAIL_TO_UNDERSTAND = "Não consegui entender o que você disse"
-
-
-class DATABASE:
-    DATABASE_URL = os.environ.get("DATABASE_URL")
-    ENGINE = create_engine(DATABASE_URL)
+    LLM_OPTIONS = ["Ollama Server", "GPT-4"]
+    EGO = """ *** Answer as briefly as possible, without writing titles 
+    or labels, in a way that doesn't sound like an AI response, but 
+    rather how a grumpy man in his 40s would answer. Everything must
+    be answered in English. If code is needed, leave it at the end of
+    the 40s-style answer and separate it with "__code__" !!! *** """
```

Comment:
deve ser parte da configuração da plataforma, o BE que deve ser responsável por isso, para que o susário controle os padrões